providers/gitlab/resources/gitlab.lr

```diff
@@ -89,7 +89,9 @@ gitlab.project @defaults("fullName visibility webURL") {
   // Approval settings for the project
   approvalSettings() gitlab.project.approvalSettings
   // Protected branches settings for the project
-  protectedBranches() []gitlab.project.repository.protectedBranch
+  protectedBranches() []gitlab.project.protectedBranch
+    // List of files in the project repository
+  listFilesInRepository() []gitlab.project.file
```

Comment:
This should be a noun not a verb so probably `projectFiles`

Reply:
Thanks, Changed